src/main/java/org/alfresco/heartbeat/HBBaseDataCollector.java

```diff
@@ -48,6 +48,12 @@
      */
     private HBDataCollectorService hbDataCollectorService;
```

Comment:
The abstract class javadoc should have a clear guidance on how to document a collector.

Reply:
To keep it simple I mentioned:
 * This class is to be extended by HeartBeat data collectors.
 * Every new collector needs to document exactly what kind of data they collecting. As good examples use one

and referenced to existing collectors as examples. Would that do it?

Reply:
Yes, and what about adding the actual _template_ for the description:
```
 <ul>
  <li>Collector ID: <b>a.collector.id</b></li>
  <li><b>Data points:</b> dataP1, dataP2 ...
      <ul>
          <li><b>dataP1:</b> data type - description</li>
          <li><b>dataP2:</b> data type - description</li>
          ...
      </ul>
  </li>
 </ul>
```
(not sure I formatted it correctly)

Reply:
good idea